include/api.php

```diff
@@ -1444,7 +1445,7 @@ function api_statuses_public_timeline($type){
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.